oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/inference/InferenceConfig.java

```diff
@@ -79,32 +85,38 @@ private InferenceConfig() {
             currentInferenceConfig = activeInferenceConfig;
             isInferenceEnabled = false;
             enricherStatus = EnricherStatus.NOOP;
+            statisticsProvider = StatisticsProvider.NOOP;
         } finally {
             lock.writeLock().unlock();
         }
     }
 
+    public static void reInitialize(NodeStore nodeStore, StatisticsProvider statisticsProvider, String inferenceConfigPath, boolean isInferenceEnabled) {
+        reInitialize(nodeStore, statisticsProvider, inferenceConfigPath, isInferenceEnabled, true);
+    }
+
+    @Deprecated
```

Comment:
Why the deprecation ? This didn't go in any public release - right ?

Reply:
Removed deprecation we can keep both constructors.